dubbo-common/src/main/java/org/apache/dubbo/common/extension/ExtensionLoader.java

```diff
@@ -521,11 +511,15 @@ private T createExtension(String name) {
             throw findException(name);
         }
         try {
+            EXTENSION_INSTANCES.computeIfAbsent(clazz, c -> {
+                try {
+                    return c.newInstance();
+                } catch (Throwable t) {
+                    throw new IllegalStateException("Extension instance(name: " + name + ", class: " +

```

Comment:
can we use this instead of string concatination.
`throw new IllegalStateException(String.format("Extension instance(name: %s,class: %s could not be instantiated: %s",name,type,t.getMessage),t);`

Reply:
I don't think it's not very worthwhile because we are using Java 8 features, and in Java 8 the string concatenation is optimized in compile time, isn't it?

Reply:
After looking at you reply I refesh my understanding, string concatination has better performance then String.format. . Only one point where String.format could be good chouce, if we want to throws localize message from resource bundle. But I think in this case it does not apply, so what I am on with you on this. No need of change for this. 
Thanks for helping out to refresh my understanding 😄